From 7304bbd5d8ad56dcf3ee9fadf74f6104c49d4df7 Mon Sep 17 00:00:00 2001
From: Giovanni Campagna <gcampagna@src.gnome.org>
Date: Fri, 12 Apr 2013 15:25:15 +0000
Subject: GVariant: fix transfer annotation

g_variant_new_from_bytes() returns a floating reference, so it
must be annotated (transfer none)

https://bugzilla.gnome.org/show_bug.cgi?id=697887
---
diff --git a/glib/gvariant-core.c b/glib/gvariant-core.c
index b96bc90..f823e01 100644
--- a/glib/gvariant-core.c
+++ b/glib/gvariant-core.c
@@ -498,7 +498,7 @@ g_variant_alloc (const GVariantType *type,
  *
  * A reference is taken on @bytes.
  *
- * Returns: a new #GVariant with a floating reference
+ * Returns: (transfer none): a new #GVariant with a floating reference
  *
  * Since: 2.36
  */
--
cgit v0.9.1
